src/scripts/main.js

```diff
@@ -1,3 +1,73 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const tBody = table.tBodies[0];
+const tableRows = table.rows;
+
+const appendRowButton = document.querySelector('.append-row');
+const removeRowButton = document.querySelector('.remove-row');
+const appendColumnButton = document.querySelector('.append-column');
+const removeColumnButton = document.querySelector('.remove-column');
+
+let td;
+let tr;
+let tdHTML = '';
+const maxCount = 10;
+const minCount = 2;
+
+appendRowButton.addEventListener('click', function() {
```

Comment:
You should use arrow functions for event listeners to ensure consistent and clean code.